xml/System.Resources.Extensions/DeserializingResourceReader.xml

```diff
@@ -155,10 +155,10 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
+        <summary>Returns an enumerator for this <see cref="T:System.Resources.Extensions.DeserializingResourceReader" /> object.</summary>
+        <returns>An enumerator for this <see cref="T:System.Resources.Extensions.DeserializingResourceReader" /> object.</returns>
         <remarks>To be added.</remarks>
```

Comment:
should we add the remarks here as we did in other PRs about EII?
I'll add that to the wiki as well.

Reply:
Can you please provide a remark suggestion for EII?

Reply:
Can we do this in a separate PR, @mairaw?

Reply:
No problem. I've added the boilerplate text to our writing guidelines here: https://github.com/dotnet/dotnet-api-docs/wiki/Remarks#explicit-interface-implementation